pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -2199,6 +2199,14 @@ public void startSendOperation(Producer producer, int msgSize, int numMessages)
             }
             isPublishRateExceeded = producer.getTopic().isBrokerPublishRateExceeded();
         } else {
+            if (producer.getTopic().isResourceGroupRateLimitingEnabled()) {
```

Comment:
It seems if we set `preciseTopicPublishRateLimitingEnable=true`, then it will disable the RG rate limiter. But this parameter doesn't provide a description of it.